internal/health/health_int_test.go

```diff
@@ -75,11 +74,13 @@ func TestService_Readiness_Integration(t *testing.T) {
 	})
 }
 
-func assertStatus(t *testing.T, statuses []*ComponentStatus, identifier string, expectedHealthy bool, expectedError string) {
+func assertStatus(t *testing.T, statuses []*ComponentStatus, identifier string, expectedHealthy bool) {
+	expectedErrorPrefix := `Get "http://invalid-host:2212/ws/v1/scheduler/healthcheck": dial tcp: lookup invalid-host`
```

Comment:
I think we should provide the error to the func as we might simulate in the future other errors.
I'd define it in the test case as `wantErrorMsg` and assert it here it contains the error message.

Reply:
> I think we should provide the error to the func as we might simulate in the future other errors. I'd define it in the test case as `wantErrorMsg` and assert it here it contains the error message.

Good suggestion - fixed in commit cf4e4dd